docs/en_US/jailbreak/using-nitotv.md

```diff
@@ -0,0 +1,71 @@
+## Required Reading
+
+::: tip
+
+This page is optional, but helpful if you are unfamiliar with jailbreaking.
+
+:::
+
+nitoTV is a package manager that has been used since the start of jailbreaking Apple TV devices. It can be used to install tweaks, utilites, tools and other essentials.
+
+![A picture of an Apple TV running nitoTV](/assets/images/nitotv_picture.png)
+
+<p><router-link to="/faq/#what-are-tweaks">Tweaks</router-link> are small modifications to change the look and/or feel of your device. You can find tweaks on repositories, colloquially referred to as "repos".</p>
+
+<p><router-link to="/faq/#what-s-a-repo">Repositories</router-link> are essentially databases that contain lots of tweaks for you to install. When you add repos to nitoTV, you will be able to install the tweaks on it, and look for them using the search function.</p>
+
+Keeping in mind, most iOS repositories DO NOT work with tvOS!
```

Comment:
I feel like this would be better worded as "Most iOS repositories **will not work** with tvOS", put into a warning box thing.